examples/lzapp-migration/README.md

```diff
@@ -93,11 +119,77 @@ More information about available CLI arguments can be found using the `--help` f
 
 ```bash
 npx hardhat lz:deploy --help
+
 ```
 
-Below is an expanded README section that describes not only the overall configuration and wiring between the V1 and V2 endpoints but also explains the purpose of the overridden tasks provided in the repository:
+### Solana (OFT202)
+
+Deploy the Solana OFT202 Program
```

Comment:
Awkward since the solana-oft example now uses anchor 1.18 explicitly in the deploy instructions #1262 .

Reply:
are you referring to the usage of `--max-len` ?